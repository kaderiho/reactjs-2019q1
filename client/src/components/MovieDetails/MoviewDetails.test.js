import React from 'react';
import { shallow } from 'enzyme';

import MoviewDetails from './MovieDetails';

/**
 * Tests:
 * 
 * - Match snapshot
 */
describe('<MoviewDetails /> component: ', () => {
    it('Should render correctly', () => {
        const wrapper = shallow(<MoviewDetails />);

        expect(wrapper).toMatchSnapshot();
    });
});
