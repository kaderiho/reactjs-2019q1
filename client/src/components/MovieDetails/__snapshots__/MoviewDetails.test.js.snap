// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoviewDetails /> component:  Should render correctly 1`] = `
<div
  className="movieDetails"
>
  <img
    className="movieDetails-image"
  />
  <div
    className="movieDetailsDescription"
  >
    <h1
      className="movieDetailsDescription-title"
    >
       
      <span
        className="movieDetailsDescription-raiting"
      />
    </h1>
    <span
      className="movieDetailsDescription-releaseDate"
    >
      NaN
    </span>
    <span
      className="movieDetailsDescription-runtime"
    >
       min
    </span>
    <p
      className="movieDetailsDescription-overview"
    />
  </div>
</div>
`;
